### ISSUE TITLE
Flow colours on day numbers

### ISSUE THREAD

**Issue by viiuan** [EmmaTellblom/Mensinator#139]
I recently started trying out this app over Drip as it has not been updated in a while (and lacks a dark mode, lol). But I like how the day number backgrounds were coloured according to the flow that day.
Could you make the flow symptoms more built-in and have the colour display on the day number? Maybe a toggle for a symptom being a flow setting if people want to add custom ones (e.g. spotting)?
This is what I mean:
![Screenshot_20240922-003920_drip](https://github.com/user-attachments/assets/c391a2a1-603b-45f1-92f5-91e8d2efb967)

**Comment by EmmaTellblom** [comment]
Welcome! This is absolutely something we can look closer at!

**Comment by carstenhag** [comment]
@viiuan Is it because you want to see light/medium/high flow in a "bigger" way, or is it mostly about the UI/looks? I feel like both are probably helpful/valid, but the look can be achieved in an easier way & there are less edge cases.

What would happen if the user has only entered flow symptoms for 1 out of 4 days? How do we handle custom colors being set (Maybe it makes sense to disallow changing the period color)?

**Comment by viiuan** [comment]
Yeah, I meant seeing it more easily. Having flow on the number leaves more room for the smaller dots underneath
Also,  if you don't enter flow in Drip, there's no background
